docs/react-types/ComponentProps.md

```diff
@@ -0,0 +1,76 @@
+---
+title: ComponentProps<Element>
```

Comment:
Do we actually handle elements and element types or just ElementType?

Reply:
That's right, should be `<ElementType>`. Updated!